hubblestack/files/hubblestack_nova/misc.py

```diff
@@ -278,6 +278,210 @@ def test_failure_reason(reason):
     '''
     return reason
 
+def test_mount_attrs(mount_name,attribute):
```

Comment:
We need to add If check as well. 
For ex: If a /tmp partition exists run the following command and verify that the nodev option is set
on /tmp.